test/svm/SvmSpoke.HandleReceiveMessage.ts

```diff
@@ -520,35 +520,35 @@ describe("svm_spoke.handle_receive_message", () => {
       messageBody,
     });
 
-    // Remaining accounts specific to EmergencyDeleteRootBundle.
+    // Remaining accounts specific to EmergencyDeletedRootBundle.
```

Comment:
I don't think this and below should be renamed as it refers to accounts, not an event